### ISSUE TITLE
Crash when switching optimization profiles on GTX 1080

### ISSUE THREAD

**Issue by Sean-PW** [nicehash/NiceHashQuickMiner#518]
In attempting to optimize my GTX 1080 I realized that changing the memory clock when memory timings are enabled is unstable.  When switching memory timings (or at least switching profiles) it may be more stable to disable all timings, then reapply memory timings.  This issue caused me to think my 1080 was unstable with the profiles when it wasn't.

More detail on timings:
https://www.reddit.com/r/NiceHash/comments/lqmprb/gtx_1080_hashrate_after_oc/gy6zo1n?utm_source=share&utm_medium=web2x&context=3

1. (use a GTX 1080 for this test -- I suspect other 10 series cards have the same issue)
2. Start QuickMiner.  
3. Enable "Efficient Low" profile in the web interface and wait a minute for it to take effect
4. Enable "Efficient" profile.
5. Issue: I got a crash pretty much 100% of the time with this workflow.
6. If I did a different workflow of clearing memory timings, setting OC settings, then setting memory timings again, it did not crash.  Even if I wasn't mining, changing the memory clock delta with the memory timings enabled caused a crash or at minimum display artifacts.

Expected Behavior: Efficient profile runs without crashing if the card accepts it

QuickMiner 0.5.1.3 Stable
Latest NVIDIA driver (466.27)
MSI GTX 1080 in PCI slot (x16) on MB
Intel 7700K

**Comment by williansti** [comment]
here the same problem happens with two gtx 1080 g1 gaming, the small solution I found was not to use a profile, do not use optimized profiles, leave the default .

ok so will you have a low hash? 19-24mh/s? use the msi afterburn and lower the power limit to 80%.

start nicehash quick, and after 5 minutes of mining perform a magic pill...... 29/30mh/s constants !

pill download = https://github.com/Virosa/ETHlargementPill/raw/master/ETHlargementPill-r2.exe

**Comment by TimSirmovics** [comment]
I noticed the same on a 1080 as well. I ended up doing a full reboot in between any change while testing profiles.

**Comment by nicehashdev** [comment]
There is nothing it can be done. Unfortunately, when it comes to 1080, each model is very unique and each one requires custom settings for max performance/stability. Unifying all these custom settings into single profiles that would work for all cards is impossible. If standard optimization profiles do not work for you, you can still use custom OCTune settings for your card.

**Comment by Sean-PW** [comment]
ok, thank you, I understand.  

Is it possible to at least disable the memory timings fully, set the new memory clock, then set the memory timings again?  I believe that would clear up many of the crashes I found in my testing.  I was getting consistent GPU crashes when I changed the memory clock with memory timings enabled, but if I reset them (by setting memory clock to 0) then set the new memory clock -> no crash.
